src/components/LaunchDetails/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, Typography } from '@mui/material';
+import './CardDetails.scss';
+
+interface CardDetailsProps {
+  values: string[];
+}
+export const CardDetails = ({ values }: CardDetailsProps) => {
+  const detailsCaptions: string[] = ['Area:', 'Status:', 'Date:'];
+
+  return (
+    <Box className="card-details">
+      {detailsCaptions.map((caption: string, index: number) => (
+        <Typography variant="body1" key={index}>
```

Comment:
do we really need typography here?

Reply:
yes, Typography provides common font styles from theme.ts like font-family, font-size, color etc.